security/pfSense-pkg-snort/files/usr/local/www/snort/snort_define_servers.php

```diff
@@ -57,8 +54,8 @@
 );
 
 /* if user has defined a custom ssh port, use it */
-if(is_array($config['system']['ssh']) && isset($config['system']['ssh']['port']))
-        $ssh_port = $config['system']['ssh']['port'];
+if(config_get_path('system/ssh/port'))
+        $ssh_port = config_get_path('system/ssh/port');
 else
         $ssh_port = "22";
```

Comment:
```suggestion
$ssh_port = config_get_path('system/ssh/port', '22');

```

Reply:
Modified to use a default return value to simplify the code.